server/swagger_v2.yml

```diff
@@ -0,0 +1,482 @@
+openapi: "3.0.0"
+info:
+  title: Gateway Service
+  version: 0.1.0
```

Comment:
This needs to be the version of chronograf and needs to be added to the release procedure.

Reply:
> and needs to be added to the release procedure.

what does this mean?

Reply:
probably to `.bumpversion.cfg` (according to docs/release.md)

@lukevmorris is bumpversion used in the release process to update the release numbers?